### PR TITLE
Layout cleanup

### PR DESCRIPTION
**Pull request by dblodorn**
1. AddressView - added href tag linking out to zora.co profile for both username & wallet address... there's no explicit styling on the link tag, functions same way it does in .co
2. refactored layout in bid history - the arrow placement is now correct, although have not tested this in create-auction-house template yet (where alan noticed the layout bug)
3. refactored creator equity layout to all be on a single row, changed the default text string to read "x% of proceeds go to"

### CODE REVIEW

**Review by iainnash — APPROVED** [review]
Looks good!

File: src/components/AddressView.tsx [hunk 1]
```diff
@@ -8,20 +9,27 @@ type AddressViewProps = {
...
+  const { getStyles } = useMediaContext();
   const username = useZoraUsername(address);
 
   const addressFirst = address.slice(0, showChars + PREFIX_ADDRESS.length);
   const addressLast = address.slice(address.length - showChars);
 
   if (username.username?.username) {
-    return <span>{`@${username.username.username}`}</span>;
+    return (
+      <a {...getStyles("addressLink")} href={`https://zora.co/${username.username.username}`} target="_blank">
```
Comment (iainnash) [inline src/components/AddressView.tsx:20]: nit: add `rel="noreferrer"` for security/seo reasons

File: tests/components/BidHistory.test.tsx [hunk 2]
```diff
@@ -138,7 +138,8 @@ describe("AuctionInfo", () => {
       );
 
       const wonArea = await screen.findByText("won the auction");
-      expect(wonArea.parentElement?.innerHTML).toContain("May 14, 10:23 PM");
+      
+      expect(wonArea.parentElement?.parentElement?.parentElement?.innerHTML).toContain("May 14, 10:23 PM");
```
Comment (iainnash) [inline tests/components/BidHistory.test.tsx:142]: Can just do `await screen.findByText("May 14, 10:23 PM");` instead.

File: tests/components/BidHistory.test.tsx [hunk 3]
```diff
@@ -153,7 +154,7 @@ describe("AuctionInfo", () => {
       );
 
       const wonArea = await screen.findByText("won the auction");
-      expect(wonArea.parentElement?.innerHTML).toContain("June 11, 12:25 PM");
+      expect(wonArea.parentElement?.parentElement?.parentElement?.innerHTML).toContain("June 11, 12:25 PM");
```
Comment (iainnash) [inline tests/components/BidHistory.test.tsx:157]: same

File: tests/components/BidHistory.test.tsx (hunk 2 above)
Comment (dblodorn) [reply tests/components/BidHistory.test.tsx]: Changed to `await screen.findAllByText("May 14, 10:23 PM");` as there are two instances of the datestamp.

File: tests/components/BidHistory.test.tsx (hunk 3 above)
Comment (dblodorn) [reply tests/components/BidHistory.test.tsx]: Changed to `await screen.findAllByText("June 11, 12:25 PM");` as there are two instances of the datestamp.

File: src/components/AddressView.tsx (hunk 1 above)
Comment (dblodorn) [reply src/components/AddressView.tsx]: Added this to all instances of `target="_blank"`

**Pull request merged**